cliquet/resource.py

```diff
@@ -154,13 +140,13 @@ def validate(self, record):
                 self.request.errors.add('body', name=field, description=error)
             raise errors.json_error(self.request.errors)
 
-    def add_timestamp_header(self, response):
+    def _add_timestamp_header(self, response):
```

Comment:
Should this stay public, in case a library user would like to add it to other endpoints?